bagelbids/tests/test_cli_bids.py

```diff
@@ -27,6 +32,43 @@ def test_bids_valid_inputs_run_successfully(
     ).exists(), "The pheno_bids.jsonld output was not created."
 
 
+@pytest.mark.parametrize(
+    "session_idx, bids_session",
+    [(0, "01"), (1, "02")],
```

Comment:
:cherries:: I don't think parameterizing the sessions is necessary here - it would be different if you ran the CLI for different datasets (e.g. `synthetic`, `ds005`, ...). We know the sessions in the dataset, so we could just make a single assertion of the form: 


```python
assert set(["ses-01", "ses-02", ...]).issubset(set([ses.label for sub in pheno_bids["hasSamples"] for ses in  sub["hasSession"]))
```

we lose the ability to assert the exact order in the json (don't think that's strictly required), but we gain not having to rerun the CLI for each session

Reply:
Sure, that works!

Reply:
Just a heads up that I modified your suggested assertion so that we're using list comprehension for each subject instead of sets. This way we can catch if 1) not all subjects have _both_ ses-01 and ses-02, like we expect based on the BIDS dir, and 2) session order in json is not as expected